requirements.txt

```diff
@@ -1 +1,2 @@
 -e git+https://github.com/f5devcentral/f5-ctlr-agent.git@671a0defbea9e7def753adb685cbd9230754f131#egg=f5-ctlr-agent
+-e git+https://github.com/f5devcentral/f5-cccl.git@cfaaa7713fddf9fe28400317548d25b104d3b865#egg=f5-cccl
```

Comment:
We specifically tried to eliminate direct dependency between k8s-bigip-ctlr and f5-cccl (and want to be only dependent on f5-ctlr-agent). Is there a reason to add this back?